common.gradle.kts

```diff
@@ -159,10 +159,13 @@ repositories {
         maven("https://maven.theillusivec4.top/")
     }
 
+    // Mixin
+    maven("https://repo.spongepowered.org/maven/")
```

Comment:
what is this needed for?

Reply:
When I updated Loom I was having an issue with Gradle finding the mixin dependencies. I added this just to be on the safe side, but it may have just been a caching issue with my local environment. I think it may be that gradle is occasionally getting hung up on a cancelled build which may cause erroneous behavior, but I figured I'd just leave the repo added just in case.

Reply:
ah I see what you mean, that's ok then